person/src/com/person/core/containers/PersonList.java

```diff
@@ -0,0 +1,46 @@
+package com.person.core.containers;
+
+import com.person.core.model.BasePerson;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PersonList {
+
+    private List<BasePerson> people = new ArrayList<>();
```

Comment:
@tatevikb why people if you are building persons?

Reply:
@nsimonyan Plural form of person is people, but I can make it persons, np.